modin/pandas/test/internals/test_repartition.py

```diff
@@ -46,15 +47,23 @@ def test_repartition(axis, dtype):
 
     if dtype == "DataFrame":
         results = {
-            None: (1, 1),
-            0: (1, 2),
-            1: (2, 1),
+            None: ([4, 0, 0, 0], [3, 0, 0, 0]),
+            0: ([4, 0, 0, 0], [2, 1]),
+            1: ([2, 2], [3, 0, 0, 0]),
         }
     else:
         results = {
-            None: (1, 1),
-            0: (1, 1),
-            1: (2, 1),
+            None: ([4, 0, 0, 0], [1]),
+            0: ([4, 0, 0, 0], [1]),
+            1: ([2, 2], [1]),
         }
```

Comment:
we will be able to revert this after #5543 is merged, right?

Reply:
reverted